src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -836,12 +836,11 @@ ServerHandlerImp::processRequest (Port const& port,
     }
     auto response = to_string (reply);
 
-    rpc_time_.notify (static_cast <beast::insight::Event::value_type> (
+    rpc_time_.notify (
         std::chrono::duration_cast <std::chrono::milliseconds> (
```

Comment:
Actually the cast isn't needed at all because `beast::insight::Event::notify` can take any `duration`.  It internally rounds up to the next `millisecond`.

I removed the `static_cast` and kept in the `duration_cast` just to preserve existing behavior.  This call rounds towards zero to the `millisecond` instead of letting `notify` round up.  If we prefer the round up, we can simply remove the `duration_cast`.